src/messages/MPGStats.h

```diff
@@ -44,6 +44,31 @@ class MPGStats final : public PaxosServiceMessage {
   void print(std::ostream& out) const override {
     out << "pg_stats(" << pg_stat.size() << " pgs seq " << osd_stat.seq << " v " << version << ")";
   }
+  void dump_stats(ceph::Formatter *f, bool with_net = true) const {
+    f->open_object_section("stats");
+
+    f->open_array_section("pg_stat");
+    for(auto& s : pg_stat) {
```

Comment:
can this be const auto&?

Reply:
Updated but IMO this is rather an overkill since the method itself is marked as const..